FirebaseAuthUI/Strings/en.lproj/FirebaseAuthUI.strings

```diff
@@ -67,18 +67,15 @@
 /* Placeholder for the password text field in a sign up form. */
 "ChoosePassword" = "Choose password";
 
-/* A notice displayed when the user is creating a new account. The first placeholder is a button label (ex. Next). The second placeholder is "Terms of Service". */
-"TermsOfServiceNotice" = "By tapping %@, you are indicating that you agree to the %@.";
-
-/* A notice displayed on the Auth Picker screen. The first placeholder is the terms of service agreement link, the second place holder is the privacy policy agreement link. */
-"TermsOfServiceAuthPicker" = "By continuing, you are indicating that you accept our %@ and %@.";
-
 /* Text linked to a web page with the Terms of Service content. */
 "TermsOfService" = "Terms of Service";
 
 /* Text linked to a web page with the Privacy Policy content. */
 "PrivacyPolicy" = "Privacy Policy";
 
+/* A message displayed when the first log in screen is displayed. The first placeholder is the terms of service agreement link, the second place holder is the privacy policy agreement link. */
+"TermsOfServiceMessage" = "By continuing, you are indicating that you accept our %@ and %@.";
+
```

Comment:
Can you submit a follow-up CL that adds the translation strings for this file? The transconsole project is named FirebaseUI.

Reply:
Ack. Will Do.